_oasis

```diff
@@ -94,7 +94,7 @@ Library cohttp_lwt
 Library cohttp_lwt_unix
   Build$:        flag(lwt_unix)
   Install$:      flag(lwt_unix)
-  Path:          lwt
+  Path:          lwt_unix
   Findlibname:   lwt
```

Comment:
I'm not changing the findlib structure of the library just moving the unix related stuff into its own directory.